src/Statement/Statement.php

```diff
@@ -45,12 +45,40 @@ class Statement extends Claim {
 	 *
 	 * @param Claim $claim
 	 * @param ReferenceList|null $references
+	 * or
+	 * @param Snak $mainSnak
+	 * @param Snaks|null $qualifiers
+	 * @param ReferenceList|null $references
+	 * @param string|null $guid
 	 */
-	public function __construct( Claim $claim, ReferenceList $references = null ) {
+	public function __construct( $claim /* , $args */ ) {
```

Comment:
Suggestion:
```php
/**
 * @since 2.0
 *
 * @param Snak|Claim $mainSnakOrClaim
 * @param Snaks|ReferenceList|null $qualifiersOrReferences
 * @param ReferenceList|null $references
 * @param string|null $guid
 */
public function __construct(
	$mainSnakOrClaim,
	$qualifiersOrReferences = null,
	ReferenceList $references = null,
	$guid = null
) {
```